application/src/Api/Adapter/ItemSetAdapter.php

```diff
@@ -61,12 +61,6 @@ public function buildQuery(QueryBuilder $qb, array $query)
         }
 
         if (!empty($query['site_id'])) {
-            $siteAdapter = $this->getAdapter('sites');
-            try {
-                $site = $siteAdapter->findEntity($query['site_id']);
-            } catch (Exception\NotFoundException $e) {
-                $site = null;
-            }
```

Comment:
Though the site isn't used here, this is intended to ensure that you cannot perform a query against a private site you don't have access to.

Reply:
So it is not clear how it works, so a comment should be added at least.